docs/docs/api.md

```diff
@@ -0,0 +1,346 @@
+---
```

Comment:
Once you rebase to master, you should be good to remove the `docs/docs` directory and use what's there already

Reply:
I think it may have to live in a folder. We can rename it to  a custom folder though. 
https://docusaurus.io/docs/en/site-config#customdocspath-string

Do you know if there is a another way to do it like ya'll want?

Reply:
Seeing how we do stuff at Jest repository, how about having this:
```
docs/
  api.md
website/
  docusaurus taking stuff from `docs/`
...rest
```

Reply:
I've moved it around. Can you take a look?